driver/driver_test.go

```diff
@@ -129,6 +134,10 @@ var _ = Describe("Contrail Network Driver", func() {
 		docker := getDockerClient()
 		_ = createValidDockerNetwork(docker)
 
+		// we need to cleanup here, because otherwise docker keeps the named pipe file open,
+		// so we can't remove it
+		cleanupAllDockerNetworksAndContainers(docker)
```

Comment:
BTW. Now that I think about it, you should probably only delete containers and networks created during the test, not all of them.